README.md

```diff
@@ -267,22 +272,22 @@ inspector.validate(schema2, c2); // Invalid: "@.lorem" is missing and not option
 
 ---------------------------------------
 
-### uniqueness
+<h3 id="v_uniqueness">uniqueness</h3>
```

Comment:
These are good changes. I think this is the best way for us to solve the problem of multiple headings with the same label (ex. "uniqueness"). We prefer the markdown syntax (`###`) but fall back to HTML when we need to like in this situation. Good work.